host/CMakeLists.txt

```diff
@@ -300,6 +331,17 @@ if (USE_LIBSGX)
 endif ()
 
 # Compile definitions and options
+target_compile_definitions(oehostverify
+  PUBLIC
+  # NOTE: This definition is public to the rest of our project's
+  # targets, but should not yet be exposed to consumers of our
+  # package.
+  $<BUILD_INTERFACE:OE_API_VERSION=2>
```

Comment:
Just a random question: when _are_ we going to make this public? This release?

Reply:
no idea.

Reply:
🤷‍♂️

Reply:
@radhikaj we should track the answer to this question.